scripts/standardize-artifacts.ts

```diff
@@ -0,0 +1,162 @@
+import yargs from "yargs";
+import path from "node:path";
+import { readdir } from "node:fs/promises";
+import {
+  readFileSync,
+  existsSync,
+  mkdirSync,
+  writeFileSync,
+  readdirSync,
+  unlinkSync,
+} from "node:fs";
+import {
+  BuildInfoInterface,
+  ArtifactInterface,
+  Dbg,
+  Contract,
+} from "./standardize-artifacts-interface";
+
+const getContracts = async (artifactsDirectory: string, dirName: string) => {
+  let contracts: Contract[] = [];
+  const items = await readdir(dirName, { withFileTypes: true });
+
+  for (const item of items) {
+    if (item.isDirectory() && item.name.indexOf(".sol") > 0) {
+      const contractName = path.parse(item.name);
+      const contract: Contract = {
+        name: contractName.base,
+        relativePath: path.join(dirName, item.name).slice(artifactsDirectory.length + 1),
+        dbg: `${contractName.name}.dbg.json`,
+        artifact: `${contractName.name}.json`,
+      };
+      contracts.push(contract);
+    } else if (item.isDirectory()) {
+      contracts = [
+        ...contracts,
+        ...(await getContracts(artifactsDirectory, `${dirName}/${item.name}`)),
+      ];
+    }
+  }
+
+  return contracts;
+};
+
+const removeFilesInDirectory = (directory: string) => {
+  for (const file of readdirSync(directory)) {
+    unlinkSync(path.join(directory, file));
+  }
+};
+
+yargs
+  .scriptName("standardizeArtifacts")
+  .recommendCommands()
+  .demandCommand(1)
+  .strict(true)
+  .showHelpOnFail(true)
+  .command(
+    "run",
+    "Generates Truffle-like artifacts from HH artifacts",
+    (args) =>
+      args
+        .option("outputDir", { type: "string", description: "Path of output directory" })
```

Comment:
Would be good to require these arguments `.requiresArg()` and provide an explanation when they are not provided `.demandOption()`

Reply:
neither of these options is required, if not specified it will just generate truffle-like-artifacts into predefined folder